omod/src/main/webapp/pages/admin/staleAdmissionRequests.gsp

```diff
@@ -0,0 +1,59 @@
+<%
+    ui.decorateWith("appui", "standardEmrPage")
+%>
+
+<script type="text/javascript" xmlns="http://www.w3.org/1999/html" xmlns="http://www.w3.org/1999/html">
+    var breadcrumbs = [
+        { icon: "icon-home", link: '/' + OPENMRS_CONTEXT_PATH + '/index.htm' },
+        { label: "${ ui.message("coreapps.app.system.administration.label") }", link: "${ ui.pageLink("coreapps", "systemadministration/systemAdministration") }" },
+        { label: "Stale Admission Requests", link: "${ ui.pageLink("pihcore", "admin/staleAdmissionRequests") }" }
+    ];
+</script>
+
+<h3>Stale Admission Requests</h3>
+
+<form id="admission-requests-search" method="post">
+    <fieldset>
+        <p>
+            Show visits with admission requests older than <input style="min-width:0%;display:inline" type="number" name="staleAdmissionRequestsThresholdInDays" value="${ staleAdmissionRequestsThresholdInDays }" /> days,
+         </p>
+         <p>
+            where the last encounter is more than <input style="min-width:0%;display:inline" type="number" name="mostRecentEncounterThresholdInDays" value="${ mostRecentEncounterThresholdInDays }" /> days ago.
+        </p>
+        <p>
+            <input type="submit" name="action" value="Search" />
+        </p>
+    </fieldset>
+</form>
+
+<% if (staleAdmissionRequests != null) { %>
+    <form id="admission-requests-close" method="post">
+        <table id="stale-admission-requests">
+            <thead>
+                <tr>
+                    <th>&nbsp;</th>
+                    <th>Patient</th>
+                    <th>Visit Start Date</th>
+                    <th>Request Date</th>
+                    <th>Latest Encounter Date</th>
+                </tr>
+            </thead>
+            <tbody>
+                <% staleAdmissionRequests.each { request -> %>
+                    <tr>
+                        <td><input type="checkbox" name="visitsToClose" value="${ request.visit.id }" checked/></td>
```

Comment:
I like defaulting to checked, since that is the primary use case (to end a bunch of admissions or requests) and originally this was going to just be task or a single-run cleanup.  Can see what Dave and Fiona think once this is merged.